Tools/Schema.NET.Tool/SchemaSourceGenerator.cs

```diff
@@ -95,174 +141,271 @@ private static string RenderClass(GeneratorSchemaClass schemaClass)
+            (index, indent, property) =>
+            $$"""
+            {{(index == 0 ? "if" : "else if")}} ("{{property.Name}}".Equals(property, StringComparison.OrdinalIgnoreCase))
+            {
+                this.{{property.Name}} = new(value);
                 success = true;
-            }}
-            else ")}
-            {{
-                result = default;
-            }}
-        }}
-        else ")}
-        {{
-            success = base.TryGetValue(property, out result);
-        }}
-
-        return success;
-    }}
-
-    /// <inheritdoc/>
-    public override bool TryGetValue(string property, out IValues result)
-    {{
-        if (string.IsNullOrWhiteSpace(property))
-        {{
-            result = default;
-            return false;
-        }}
-
-        var success = false;
-        {SourceUtility.RenderItems(allProperties, property => $@"if (""{property.Name}"".Equals(property, StringComparison.OrdinalIgnoreCase))
-        {{
-            result = (IValues)this.{property.Name};
-            success = true;
-        }}
-        else ")}
-        {{
-            success = base.TryGetValue(property, out result);
-        }}
-
-        return success;
-    }}
-
-    /// <inheritdoc/>
-    public bool Equals({schemaClass.Name} other)
-    {{
-        if (other is null)
-        {{
-            return false;
-        }}
-
-        if (ReferenceEquals(this, other))
-        {{
-            return true;
-        }}
-
-        return this.Type == other.Type{SourceUtility.RenderItems(allProperties, property => $@" &&
-            this.{property.Name} == other.{property.Name}")} &&
-            base.Equals(other);
-    }}
-
-    /// <inheritdoc/>
-    public override bool Equals(object obj) => this.Equals(obj as {schemaClass.Name});
-
-    /// <inheritdoc/>
-    public override int GetHashCode() => HashCode.Of(this.Type){SourceUtility.RenderItems(allProperties, property => $@"
-        .And(this.{property.Name})")}
-        .And(base.GetHashCode());
-}}";
+            }
+            """,
+            indent,
+            SourceDelimeter.NewLine);
+
+        return SourceUtility.Render(
+            $$"""
+            /// <inheritdoc/>
+            public override bool TrySetValue(string property, IEnumerable<object> value)
+            {
+                if (string.IsNullOrWhiteSpace(property))
+                {
+                    return false;
+                }
+
+                bool success;
+            {{conditions}}
+                else
+                {
+                    success = base.TrySetValue(property, value);
+                }
+
+                return success;
+            }
+            """,
+            indent);
     }
 
-    [SuppressMessage(
-        "StyleCop.CSharp.LayoutRules",
-        "SA1513:Closing brace should be followed by blank line",
-        Justification = "Interpolated string")]
+    private static string RenderClassTryGetVaue1(GeneratorSchemaProperty[] allProperties, int indent)
```

Comment:
Typo in method name - thinking maybe it would be better called `RenderClassTryGetValue_OneOrMany`.

Reply:
Done